tests/foreman/api/test_docker.py

```diff
@@ -453,9 +452,31 @@ def test_add_multiple_docker_repos_to_content_view(self):
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        upstream_name = u'busybox'
```

Comment:
If the goal is perfect readability, then we should just not call `_create_repository()` at all. Let's either use the function or not. Half-using the function is weird.